File: Course-Project/app/src/main/java/com/vladislavmyasnikov/courseproject/data/models/TaskInfo.java
package com.vladislavmyasnikov.courseproject.data.models;

import com.google.gson.annotations.SerializedName;

public class TaskInfo {

    @SerializedName("id")
    public int id;

    @SerializedName("status")
    public String status;

    @SerializedName("mark")
    public double mark;

    @SerializedName("task")
    public Task task;
}

File: Course-Project/app/src/main/java/com/vladislavmyasnikov/courseproject/data/models/TaskInfo.kt
package com.vladislavmyasnikov.courseproject.data.models

import com.google.gson.annotations.SerializedName

class TaskInfo {

    @SerializedName("id")
    var id: Int = 0

    @SerializedName("status")
    var status: String? = null

    @SerializedName("mark")
    var mark: Double = 0.toDouble()

    @SerializedName("task")
    var task: Task? = null
}
